setup.py

```diff
@@ -55,7 +56,7 @@
     'cc-plugin-imos>=1.3.0',
     'matplotlib==1.5.1',
     'aodntools>=0.2.2',
-    'pandas==0.22.0'
+    'pandas==0.24.2'
```

Comment:
Since it depends on a new aodncore feature, should also bump the required aodncore version as well to `>=0.31.0`

Reply:
Ah yes, good pick-up! Thanks. Will update.

Reply:
Done